components/brave_sync/brave_sync_service_impl.cc

```diff
@@ -603,10 +611,13 @@ void BraveSyncServiceImpl::SendDeviceSyncRecord(
 }
 
 static const int64_t kCheckUpdatesIntervalSec = 60;
+static const int64_t kCheckInitialUpdatesIntervalSec = 1;
 
-void BraveSyncServiceImpl::StartLoop() {
+void BraveSyncServiceImpl::StartLoop(const bool initial_frequent_updates) {
   timer_->Start(FROM_HERE,
-                  base::TimeDelta::FromSeconds(kCheckUpdatesIntervalSec),
+                  base::TimeDelta::FromSeconds(
+                    initial_frequent_updates ? kCheckInitialUpdatesIntervalSec :
```

Comment:
s/initial_updates/waiting_for_second_device/ because that is the only case we fetch every 1 second

Reply:
thanks, fixed.